packages/gluestick-generators/src/index.js

```diff
@@ -0,0 +1,6 @@
+/* @flow */
+
+module.exports = {
+  default: require('./generator'),
+  parseConfig: require('./parseConfig'),
+};
```

Comment:
It's also possible to export some function as a default, and other when using dot:
```
module.exports = () => '1';
module.exports.fn = () => '2';
```
so requiring will be:
```
const fnDefault = require('some-file');
const fnOther = require('some-file').fn;
```
but I'm fine with object also.